### DISCUSSION TITLE
[Feature] Set in the Storage Template the Location

### DISCUSSION THREAD

**Discussion by UNRA1DUser** [immich-app/immich#8303] [Feature Request]
### The feature

Hi,

It would be interesting to set the Location (like in the Map) in the Storage Template to create a folder structure like

{{album}}/**{{location}}**/{{y}}/{{y}}-{{MM}}-{{dd}}/{{filename}}
UPLOAD_LOCATION/USERNAME/Album Name/Location/2022/2022-02-03/IMAGE_56437.jpg

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by rlndjnk** [comment]
The support of the "granularity" of the search function would be perfect: variables for country, province, city

This would really be very helpful to build a directory structure that enables one to find assest based on the question "when have I been where?".

**Comment by nbcjr** [comment]
i second this. osxphotos is an FOSS software that has this, getting info from EXIF, so maybe it's easy to just port
